portal-web/test/functional/com/liferay/portalweb/macros/Form.macro

```diff
@@ -721,10 +721,17 @@ definition {
 			value1 = "Draft");
 	}
 
+	macro viewSelectFromListOnPreview {
+		AssertElementPresent(
+			key_fieldName = "${fieldName}",
```

Comment:
@ewertonluna, let's rename to keep a naming consistency:

- `viewSelectFromListOnPreview` to `viewSelectFromListOptionOnPreview`
- `key_fieldName` to `key_optionValue`
- `fieldName` to `optionValue`